ArduPlane/servos.cpp

```diff
@@ -902,6 +903,26 @@ void Plane::servos_output(void)
     quadplane.tailsitter_output();
     quadplane.tiltrotor_bicopter();
 
+     // pull up tilts and set throttle_min if LANDING_FLARE RCx_OPTION switch is set and in FW mode, manual throttle
+     if (!quadplane.in_transition() && !control_mode->is_vtol_mode() && !auto_throttle_mode && plane.channel_throttle->norm_input() < -0.95 && plane.flare_switch_active) {
+         int32_t tilt = -4500;
+         if (quadplane.tilt.tilt_type == QuadPlane::TILT_TYPE_BICOPTER) {
+                 tilt = 0;
+         }
+         if (quadplane.is_tailsitter()) {
+                 tilt = 4500;
+         }
+         SRV_Channels::set_output_scaled(SRV_Channel::k_motor_tilt, tilt);
+         SRV_Channels::set_output_scaled(SRV_Channel::k_tiltMotorLeft, tilt);
+         SRV_Channels::set_output_scaled(SRV_Channel::k_tiltMotorRight, tilt);
+         float throttle_min = aparm.throttle_min.get(); //allows ICE to run if used
+             if (arming.is_armed()) {  //prevent running motors if unarmed
+                 SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, throttle_min);
+                 SRV_Channels::set_output_scaled(SRV_Channel::k_throttleLeft, throttle_min);
+                 SRV_Channels::set_output_scaled(SRV_Channel::k_throttleRight, throttle_min);
+             }
+     }
```

Comment:
would be nice to have this in its own little function. like `servos_twin_engine_mix()`

Reply:
done